commons/src/main/kotlin/com/simplemobiletools/commons/compose/screens/ManageBlockedNumbersScreen.kt

```diff
@@ -0,0 +1,528 @@
+                    scrollBehavior,
+                    statusBarColor,
+                    colorTransitionFraction,
+                    contrastColor,
+                    onAdd,
+                    onImportBlockedNumbers,
+                    onExportBlockedNumbers
+                )
+            }
+        },
+    ) {
+        val state = rememberCanPerformVerticalScrollLazyListState()
+        val autoScrollSpeed = remember { mutableFloatStateOf(0f) }
+        LaunchedEffect(autoScrollSpeed.floatValue) {
+            if (autoScrollSpeed.floatValue != 0f) {
+                while (isActive) {
+                    state.scrollBy(autoScrollSpeed.floatValue)
+                    delay(10)
+                }
+            }
+        }
+
+        LazyColumn(
+            state = state,
+            modifier = Modifier.dragHandler(
+                lazyListState = state,
+                haptics = LocalHapticFeedback.current,
+                selectedIds = selectedIds,
+                autoScrollSpeed = autoScrollSpeed,
+                autoScrollThreshold = with(LocalDensity.current) { 40.dp.toPx() }
+            )) {
+            item(key = "unknown") {
+                SettingsCheckBoxComponent(
+                    title = if (isDialer) stringResource(id = R.string.block_not_stored_calls) else stringResource(id = R.string.block_not_stored_messages),
+                    initialValue = isBlockUnknownSelected,
+                    onChange = onBlockUnknownSelectedChange,
+                    modifier = startingPadding,
+                )
+            }
+
+            item(key = "hidden") {
+                SettingsCheckBoxComponent(
+                    title = if (isDialer) stringResource(id = R.string.block_hidden_calls) else stringResource(id = R.string.block_hidden_messages),
+                    initialValue = isHiddenSelected,
+                    onChange = onHiddenSelectedChange,
+                    modifier = startingPadding,
+                )
+            }
+            item(key = "divider") {
+                SettingsHorizontalDivider()
+            }
+            when {
+                !hasGivenPermissionToBlock -> {
+                    noPermissionToBlock(setAsDefault = setAsDefault)
+                }
+
+                blockedNumbers.isEmpty() -> {
+                    emptyBlockedNumbers(addABlockedNumber = onAdd)
+                }
+
+                hasGivenPermissionToBlock && blockedNumbers.isNotEmpty() -> {
+                    items(blockedNumbers, key = { blockedNumber -> blockedNumber.id }) { blockedNumber ->
+                        val isSelected = selectedIds.value.contains(blockedNumber.id)
+                        BlockedNumber(
+                            modifier = Modifier
+                                .animateItemPlacement()
+                                .semantics {
+                                    if (!isInActionMode) {
+                                        onLongClick("select") {
```

Comment:
whats this "select"?

Reply:
Long press select swipe up/down

Reply:
but what does "select" mean? Is it some command recognized by the system? Shouldnt it be some constant?

Reply:
it's just a label for the `semantics`